packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -236,6 +255,16 @@ export const rpcURLs: { [chainId: number]: string } = {
   })
 }
 
+export const rpcURLs: { [chainId: number]: string } =
+  process.env.NODE_ENV !== 'production' || process.env.NEXT_PUBLIC_IS_E2E_TEST
```

Comment:
if process.env.NEXT_PUBLIC_IS_E2E_TEST is set to false this will evaluate to true, should check more explicitly

Reply:
not in this PR

Reply:
As discussed on Slack, this condition is used in multiple places of the codebase

I created a Linear ticket to clean them up
FS-1251